packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -262,7 +267,13 @@ export class FeatureDevController {
                     ],
                 })
                 break
+            case PromptRefusalException.name:
+                this.messenger.sendErrorMessage(errorMessage, message.tabID, 0, session?.conversationIdUnsafe, true)
```

Comment:
where is the code that sets `errorMessage` ? Does it use `getTelemetryReasonDesc` ? If the error message is sent to telemetry, `getTelemetryReasonDesc` should be used somewhere in the codepath to ensure that PII is not sent to telemetry. https://github.com/aws/aws-toolkit-vscode/blob/7740f3af6872d2d9b6ea98653ef46061e0b3bdbe/packages/core/src/shared/errors.ts#L397

Reply:
No, as far I've checked in this latest version, we're not sending this errorMessage to telemetry. We can, but then I agree with you that we need sanitize PII before doing that.